.vscode/settings.json

```diff
@@ -8,5 +8,5 @@
         "test/examples",
         "packages"
     ],
-    "editor.formatOnSave": true
-}
+    // "editor.formatOnSave": true
```

Comment:
Probably need to update fantomas to 5.x because otherwise this seems to break formatting on test files.